test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -89,35 +89,37 @@ var _ = BeforeSuite(func() {
 		ClusterDefinition:  csInput,
 		ExpandedDefinition: csGenerated,
 	}
-	masterNodes, err := node.GetByRegexWithRetry("^k8s-master-", 3*time.Minute, cfg.Timeout)
```

Comment:
you probably want to review this with https://github.com/Azure/aks-engine/pull/2187/files?w=1